packages/playground/src/weblets/micro_vm.vue

```diff
@@ -40,35 +40,12 @@
 
         <RootFsSize :cpu="cpu" :memory="memory" v-model.number="rootFsSize" />
 
-        <input-validator
-          :value="cpu"
-          :rules="[
-            validators.required('CPU is required.'),
-            validators.isInt('CPU must be a valid integer.'),
-            validators.min('CPU min is 1 cores.', 1),
-            validators.max('CPU max is 32 cores.', 32),
-          ]"
-          #="{ props }"
-        >
-          <input-tooltip tooltip="The number of virtual cores allocated to your instance.">
-            <v-text-field label="CPU (vCores)" type="number" v-model.number="cpu" v-bind="props" />
-          </input-tooltip>
-        </input-validator>
-
-        <input-validator
-          :value="memory"
-          :rules="[
-            validators.required('Memory is required.'),
-            validators.isInt('Memory must be a valid integer.'),
-            validators.min('Minimum allowed memory is 256 MB.', 256),
-            validators.max('Maximum allowed memory is 256 GB.', 256 * 1024),
-          ]"
-          #="{ props }"
-        >
-          <input-tooltip tooltip="The amount of RAM (Random Access Memory) allocated to your instance.">
-            <v-text-field label="Memory (MB)" type="number" v-model.number="memory" v-bind="props" />
-          </input-tooltip>
-        </input-validator>
+        <SelectSolutionFlavor
+          :minimum="{ cpu: 1, memory: 1024 * 1, disk: 25 }"
+          :standard="{ cpu: 2, memory: 1024 * 4, disk: 50 }"
+          :recommended="{ cpu: 4, memory: 1024 * 4, disk: 100 }"
+          v-model="solution"
```

Comment:
please update the `weblet-layout` props with the solution
no need to use the cpu, memory, and disk refs now,

at the top of the weblet update the props to be like this code from dicourse weblet
```
<template>
  <weblet-layout
    ref="layout"
    :cpu="solution?.cpu"
    :memory="solution?.memory"
    :disk="(solution?.disk ?? 0) + rootFs(solution?.cpu ?? 0, solution?.memory ?? 0)"
    title-image="images/icons/discourse.png"
  >
```